lib/dynflow/action/with_sub_plans.rb

```diff
@@ -26,16 +26,20 @@ def run(event = nil)
     end
 
     def initiate
-      sub_plans = create_sub_plans
-      sub_plans = Array[sub_plans] unless sub_plans.is_a? Array
+      output.update(:total_count   => 0,
```

Comment:
Do we need to reset the numbers here? It would break the `done?` method when called sooner than any planning happens. When we go with `planned_count` number, we should not need to reset this numbers at the beginning (just initiating the `planned_count` in the `WithBulkSubPlans`